src/renderers/webgl/WebGLPrograms.js

```diff
@@ -153,7 +154,8 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 			matcap: !! material.matcap,
 			envMap: !! envMap,
 			envMapMode: envMap && envMap.mapping,
-			envMapCubeUV: ( !! envMap ) && ( ( envMap.mapping === CubeUVReflectionMapping ) || ( envMap.mapping === CubeUVRefractionMapping ) ),
+			envMapCubeUV: useCubeUV,
```

Comment:
Seems like this one is not used anywhere?

Reply:
I didn't change it, just moved the logic  up. It  seems to tigger the `GL_OES_standard_derivatives` extension.

Reply:
Can we use the newly added `cubeUVHeight` to trigger the extension instead?

Reply:
sure thing